frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -24,7 +27,13 @@ export const handle = {
 
 export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
   const csrfToken = String(session.get('csrfToken'));
-  return json({ csrfToken });
+  const lookupService = getLookupService();
+  const t = await getFixedT(request, handle.i18nNamespaces);
+  const clientStatusList = await lookupService.getAllClientFriendlyStatuses();
+
+  const meta = { title: t('gcweb:meta.title.template', { title: t('status:page-title') }) };
```

Comment:
**Massive nitpick:** Try to declare variables close to where they are being used. 

Here, `t` is being declared before `clientStatusList` but not being used until we initialize `meta`.

```suggestion
  const lookupService = getLookupService();
  const clientStatusList = await lookupService.getAllClientFriendlyStatuses();

  const t = await getFixedT(request, handle.i18nNamespaces);
  const meta = { title: t('gcweb:meta.title.template', { title: t('status:page-title') }) };
```

Reply:
@nicholas-ly, @shawn320's following other pages pattern.

Reply:
I will resolve then.